Build/linq2db.LINQPad.nuspec

```diff
@@ -11,26 +11,27 @@
 		<readme>README.md</readme>
 		<dependencies>
 			<group targetFramework=".NETCoreApp3.1">
-				<dependency id="LINQPad.Reference"                         version="1.1.0"     />
-				<dependency id="linq2db"                                   version="3.6.0"     />
+				<dependency id="LINQPad.Reference"                         version="1.3.0"     />
+				<dependency id="linq2db"                                   version="4.0.0"     />
 
-				<dependency id="Humanizer.Core"                            version="2.13.14"   />
-				<dependency id="CodeJam"                                   version="4.0.1"     />
+				<dependency id="Humanizer.Core"                            version="2.14.1"    />
+				<dependency id="CodeJam"                                   version="4.1.0"     />
 				<dependency id="System.Configuration.ConfigurationManager" version="6.0.0"     />
-				<dependency id="Microsoft.CodeAnalysis.CSharp"             version="4.0.1"     />
-				<dependency id="System.Text.Json"                          version="5.0.2"     />
-				<dependency id="dotMorten.Microsoft.SqlServer.Types"       version="1.3.0"     />
+				<dependency id="Microsoft.CodeAnalysis.CSharp"             version="4.2.0"     />
+				<dependency id="System.Text.Json"                          version="6.0.4"     />
+				<dependency id="dotMorten.Microsoft.SqlServer.Types"       version="2.5.0"     />
 
-				<dependency id="FirebirdSql.Data.FirebirdClient"           version="8.5.4"     />
-				<dependency id="MySqlConnector"                            version="2.0.0"     />
+				<dependency id="FirebirdSql.Data.FirebirdClient"           version="9.0.1"     />
+				<dependency id="MySqlConnector"                            version="2.1.10"    />
 				<dependency id="AdoNetCore.AseClient"                      version="0.19.2"    />
-				<dependency id="System.Data.SQLite.Core"                   version="1.0.115.5" />
+				<dependency id="System.Data.SQLite.Core"                   version="1.0.116"   />
 				<dependency id="System.Data.SqlClient"                     version="4.8.3"     />
 				<dependency id="System.Data.Odbc"                          version="5.0.0"     />
 				<dependency id="System.Data.OleDb"                         version="5.0.0"     />
-				<dependency id="Npgsql"                                    version="6.0.0"     />
-				<dependency id="Oracle.ManagedDataAccess.Core"             version="3.21.4"    />
-				<dependency id="IBM.Data.DB2.Core"                         version="3.1.0.500" />
+				<dependency id="Npgsql"                                    version="6.0.4"     />
+				<dependency id="Oracle.ManagedDataAccess"                  version="21.6.1"    />
```

Comment:
nuspec is for linqpad 6/7 targeting .net core. No need for netfx Oracle.ManagedDataAccess nuget